README.md

```diff
@@ -52,7 +52,7 @@ for more background, objectives, key use cases.
     npm start
     ```
 
-2. Visit the `test.html` page at [localhost:6060/test.html](http://localhost:6060/test.html)
+2. Visit the `test.html` page at [localhost:6060] (http://localhost:6060)
```

Comment:
I'd just remove the v confusing `test.html` reference.

Reply:
hahaha yeah I am inclined to agree